Formula/sphinx.rb

```diff
@@ -14,6 +14,7 @@ class Sphinx < Formula
 
   depends_on "mysql"
   depends_on "openssl"
+  depends_on "unixodbc"
```

Comment:
The default according to https://dev.mysql.com/doc/mysql-sourcebuild-excerpt/8.0/en/source-configuration-options.html is to build with unixodbc support off. Was this added to fix the build issue since it would also require `WITH_UNIXODBC=1` to make it include the functionality to support the dependency would it not ?

Reply:
Actually looking back at the build error it has nothing to do with `mysql` but rather the `sphinx` formula which probably needs a revision bump to make it work properly.

Reply:
This is test error
```
22:13:20 ==> /usr/local/Cellar/sphinx/2.2.11_1/bin/searchd --help
22:13:20 dyld: Library not loaded: /usr/local/opt/unixodbc/lib/libodbc.2.dylib
22:13:20   Referenced from: /usr/local/Cellar/sphinx/2.2.11_1/bin/searchd
22:13:20   Reason: image not found
22:13:20 Error: sphinx: failed
```

Reply:
It has been added because of this error.
We don't need a version bump, we have latest public sphinx release.
I think it should be correct to make one more PR to add unixodbc to sphinx.

Reply:
I referred to a revision bump, not a version bump which is different. Sphinx seems to autodetect the presence of unixodbc so I'm not sure how it made it in there in to begin with. If I compile locally with `unixodbc` removed it doesn't link to it so that's why I'm saying a revision bump of `sphinx` might be sufficient to remove the unixodbc linkage from sphinx and solve the issue.